consent-ws/src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -27,23 +27,26 @@
 
     @SqlQuery("SELECT c.* " +
             "FROM consents c INNER JOIN consentassociations cs ON c.consentId = cs.consentId "+
-            "WHERE cs.objectId = :datasetId")
-    Consent findConsentFromDatasetID(@Bind("datasetId") String datasetId);
+            "WHERE cs.dataSetId = :datasetId")
+    Consent findConsentFromDatasetID(@Bind("datasetId") Integer datasetId);
```

Comment:
why is the datasetId now an integer? is this now representing the DUOS_000001 id?

Reply:
No, we associated consent association with dataset. datasetId is the id of dataset where we save the rest of information related to dataset like object id, name, etc.

Reply:
we created a new field on dataset called alias where we save a number, dataset UID  is generated by the system, Moran asked to set some specific values for the first and second datasets with specific name, and the rest should be incremental

Reply:
Greg this cases DOID_001, DOID_0001, DOID_00001,  wont happen because the story says that we have specific pattern for this.

"We want to assign the DatasetID 'DUOS-000001' to a dataset which has not yet been uploaded to DUOS, but will be uploaded shortly.
We want to assign the DatasetID 'DUOS-000002' to the Melanoma_Regev dataset already in the Dataset Catalog in DUOS.
Please do not assign either of these UIDs to other datasets.
The remaining datasets already in the Dataset Catalog can be assigned the following DatasetID UIDs 'DUOS-000003', 'DUOS-000004' etc.